src/App.jsx

```diff
@@ -1,7 +1,21 @@
 import ReactDOM from "react-dom/client";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import AddPage from "../pages/AddPage/AddPage";
+import AllPage from "../pages/AllPage/AllPage";
+import CompletedPage from "../pages/CompletedPage/CompletedPage";
+import UnCompletedPage from "../pages/UnCompletedPage/UnCompletedPage";
+import ConfigurationPage from "../pages/ConfigurationPage/ConfigurationPage";
 
 const App = () => {
-
+  <BrowserRouter>
+    <Routes>
```

Comment:
If someone opens your website. They directly open `/`. It means that nothing will be loaded for them, Define a `/` route and load Either `Add` or `All` component.

Reply:
Done.